tests/BinaryFormatterTests/WhenWorkingWith_Classes.cs

```diff
@@ -55,5 +146,31 @@ public void CanWorkWith_ClasseWithCustomCtor_WithProperties_WithPublicSetter()
             Assert.Equal(before.Int, after.Int);
             Assert.Equal(before.Double, after.Double);
         }
+
+        [Fact]
+        public void CanWorkWith_ClassesWithoutCustomCtor_WithVirtualProperties_WithPublicSetter()
```

Comment:
@lukasz-pyrzyk Understood. What, if I I will named test like "CanWorkWith_WithVirtualProperties". I think, that it will be enought. Ok?